api/src/indigo_internal.h

```diff
@@ -82,6 +83,10 @@ class DLLEXPORT IndigoObject
         REACTION_ITER,
         REACTION_MOLECULE,
         GROSS_MOLECULE,
+        JSON_LOADER,
+        JSON_SAVER,

```

Comment:
IMHO,`JSON_LOADER` and `JSON_SAVER` and corresponding classes, if referring to loader and saver functionality, do not have place in the model of indigoObject types hierarchy. 
Note: The same is true regarding to `SDF_LOADER` and `SDF_SAVER` and corresponding classes, but they are out of scope of this change.

Anyway, once you rejected `class IndigoJSONLoader` inherited from `class IndigoObject`, these enums should be rejected too.

Reply:
Don't quite get the comment. Why the SDF_LOADER and SDF_SAVER (and same for JSON) are not part of IndigoObject?

Reply:
I would qualify them in another branch of hierarchy not to mix with objects->molecules->etc. 
But, yeah, now I see that you built yet another OO model, where everything is derived from IndigoObject. 
So forget the generic part of the comment.
Just we have to maintain correspondence between the enums and the existing Object classes. No class - no enum.